doc/content/lorawan/regional-limitations-of-rf-use/_index.md

```diff
@@ -0,0 +1,63 @@
+
+Usually, the limitations of sub-band K, L, M, N and Q are applied for uplink transmissions and the limitations of sub-band P are mainly applied for downlink transmissions. There are normally many more end device thatn gateways on a LoRaWAN network. Gateways can therefore run into their duty cycle limit quicker than end devices. LoRaWAN's RX2 channel is normally chose to fall in sub-bad P to benefit from the higher duty cycle limit, and provide 10x more downlinks to gateways than to end devices to combat this imbalance.
+
+By default, the RX1 receive window for downlinks uses the same frequency as the preceding uplink, while the RX2 receive window uses a fixed frequency, with the default being 869.525 MHz at DR 0 using SF12 with 125 kHz.
+
+The default frequency for beacon broadcasts and ping-slot transmissions for Class B end devices in Europe is 869.525 MHz. For LoRaWAN relays, channel 0 uses 865.1 MHz and 865.5 MHz for WOR (Wake On Radio) transmissions respectively, and channel 1 uses 865.3 MHz and 865.9 MHz for WOR ACK (Wake On Radio Acknowledgement) transmissions respectively.
+
+# US (902-928 MHz)
+
+The following regulations are applied to the USA, Canada, and all other countries in ITU Region 2 adopting the entire [FCC](https://www.fcc.gov/) (Federal Communications Commission) 47 CFR Part 15 regulations in the 902-928 ISM band.
+
+In the United States, the ISM band is divided into **80 channels**: 72 for uplink and 8 for downlink.
+
+- There are **64 uplink channels**, numbered from 0 to 63, each with a bandwidth of 125 kHz. These channels start at a center frequency at 902.3 MHz and increment linearly by 200 kHz up to 914.9 MHz.
+- There are **8 uplink channels**, numbered from 64 to 71, each with a bandwidth of 500 kHz. These channels start at a center frequency at 903.0 MHz and increment linearly by 1.6 MHz up to 914.2 MHz.
+- There are **8 downlink channels**, numbered from 0 to 7, each with a bandwidth of 500 kHz. These channels start at a center frequency at 923.3 MHz and increment linearly by 600 kHz up to 927.5 MHz.
+
+The end devices must have the capability to store parameters for the 72 uplink channels mentioned above.
+
+The dwell time for LoRaWAN in the US902-928 band refers to the amount of time that a device spends on a specific channel before switching to the next channel. The dwell time is an important parameter for LoRaWAN devices as it helps to ensure that the devices are able to successfully transmit data while avoiding interference with other devices.
+
+For the US ISM band, the [dwell time](https://www.thethingsindustries.com/docs/reference/glossary/) for transmitting a LoRaWAN message must not exceed 400 ms. It is important to consider this dwell time limitation when designing and operating devices. For example, if a device is transmitting data at a high rate, it may need to use a faster hopping rate in order to comply with the 400 ms dwell time limitation.
```

Comment:
if you want to link something on dwell time, then please do it on the first occurrence of that term